tasks/process_agent.py

```diff
@@ -127,7 +127,7 @@ def build_dev_image(ctx, image=None, push=False, base_image="datadog/agent:lates
         ctx.run(f"chmod 0444 {docker_context}/*.o {docker_context}/*.c {docker_context}/co-re/*.o")
         ctx.run(f"cp /opt/datadog-agent/embedded/bin/clang-bpf {docker_context}")
         ctx.run(f"cp /opt/datadog-agent/embedded/bin/llc-bpf {docker_context}")
-        ctx.run(f"cp pkg/network/protocols/tls/java/agent-usm.jar {docker_context}")
+
```

Comment:
I think we should undo the change - this task (and the Dockerfile) you changed are meant to patch a dev-image via a manual command line. So the change has no impact on the actual artifacts we provide for customers, and it is still required for the testing purposes